risc0/zkvm/prove/make-id/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "risc0-make-id"
-version = "0.11.0-rc.4"
+version = "0.11.0"
```

Comment:
Should this one be 0.11.1 rather than .0?

Reply:
Good catch